grid_map_pcl/CMakeLists.txt

```diff
@@ -100,38 +84,78 @@ install(
   TARGETS
     ${PROJECT_NAME}
     grid_map_pcl_loader_node
-  ARCHIVE DESTINATION ${CATKIN_PACKAGE_LIB_DESTINATION}
-  LIBRARY DESTINATION ${CATKIN_PACKAGE_LIB_DESTINATION}
-  RUNTIME DESTINATION ${CATKIN_PACKAGE_BIN_DESTINATION}
+  ARCHIVE DESTINATION lib
+  LIBRARY DESTINATION lib
+  RUNTIME DESTINATION lib/${PROJECT_NAME}
 )
 
 # Mark cpp header files for installation
 install(
   DIRECTORY include/${PROJECT_NAME}/
-  DESTINATION ${CATKIN_PACKAGE_INCLUDE_DESTINATION}
+  DESTINATION include/${PROJECT_NAME}
   FILES_MATCHING PATTERN "*.hpp"
 )
 
+install(
+  DIRECTORY
+    config
+    launch
+  DESTINATION share/${PROJECT_NAME}
+)
+
+install(
+  DIRECTORY test/test_data/
+  DESTINATION share/${PROJECT_NAME}/test_data/
```

Comment:
Do we really want to install test data in the binaries?

Reply:
Currently using `ament_index_cpp::get_package_share_directory` according to this [answer](https://answers.ros.org/question/288501/ros2-equivalent-of-rospackagegetpath/) to locate the test data, that's why test_data is being installed. Is there a different way?

Reply:
and that's needed to run the test in CI? Maybe put this install macro in the `if testing`?

Reply:
Yes, needed to run tests. Installed in `if testing`.